src/App.js

```diff
@@ -1,10 +1,35 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Home } from "./pages/Home/Home";
+import { Cart } from "./pages/Cart/cart";
+import { ProductListing } from "./pages/ProductListing/ProductListing";
+import { Wishlist } from "./pages/Wishlist/wishlist";
+import { Login } from "./pages/Auth/login/login";
+import { Signup } from "./pages/Auth/signup/signup";
```

Comment:
Put them into single  ```index.js``` and then import in single line as it's going to increase code lines into your code